src/Symfony/Bridge/Doctrine/ContainerAwareEventManager.php

```diff
@@ -136,16 +149,47 @@ public function removeEventListener($events, $listener)
         }
     }
 
+    /**
+     * Adds an EventSubscriber by its identifier in the ServiceLocator injected in the constructor.
+     */
+    public function addEventSubscriberId(string $subscriber): void
+    {
+        $this->initializedSubscribers = false;
+        $this->subscribers[$subscriber] = $subscriber;
+    }
+
+    /**
+     * Removes an EventSubscriber from its identifier in ServiceLocator injected in the constructor.
+     */
+    public function removeEventSubscriberId(string $subscriber): void
```

Comment:
should be removed, since we don't use it

Reply:
done